File: Pokémon 3D Server/Pokémon 3D Server/Class/Setting/Data List/OperatorList.vb
Public Class OperatorList
    Public Property Name As String
    Public Property GameJoltID As Integer
    Public Property OperatorReason As String
    Public Property OperatorLevel As Integer

    Public Sub New(ByVal Name As String, ByVal GameJoltID As Integer, ByVal OperatorReason As String, ByVal OperatorLevel As Integer)
        Me.Name = Name
        Me.GameJoltID = GameJoltID
        Me.OperatorReason = OperatorReason
        Me.OperatorLevel = OperatorLevel
    End Sub
End Class

File: Pokemon.3D.Server.Client/Pokemon.3D.Server.Core/Server_Client_Listener/Settings/Data/OperatorList.cs
namespace Pokemon_3D_Server_Core.Server_Client_Listener.Settings.Data
{
    /// <summary>
    /// Class containing OperatorList data
    /// </summary>
    public class OperatorList : CommonData
    {
        /// <summary>
        /// Get/Set OperatorList Level
        /// </summary>
        public int OperatorLevel { get; set; }

        /// <summary>
        /// New OperatorList
        /// </summary>
        /// <param name="Name">Player Name</param>
        /// <param name="GameJoltID">Player GameJolt ID</param>
        /// <param name="Reason">OperatorList Reason</param>
        /// <param name="OperatorLevel">OperatorList Level</param>
        public OperatorList(string Name, int GameJoltID, string Reason, int OperatorLevel)
        {
            this.Name = Name;
            this.GameJoltID = GameJoltID;
            this.Reason = Reason;
            this.OperatorLevel = OperatorLevel;
        }
    }
}
